src/components/accordion/accordion-animated-panel.tsx

```diff
@@ -0,0 +1,53 @@
+import { animated, config, useSpring } from 'react-spring';
+import * as ReachAccordion from '@reach/accordion';
+import styled from 'styled-components';
+import { useAccordionItemContext } from '@reach/accordion';
+import { useDivHeight } from './use-div-height';
+import React from 'react';
+
+const AnimatedAccordionPanel = animated(ReachAccordion.AccordionPanel);
```

Comment:
This is a wrapper coming from react spring and works like styled components as it extends the capabilities of the target component but otherwise doesn't change it. 


> You need the animation-primitive itself, and a special factory called animated. This library animates outside React (for performance reasons). Your view has to know how to handle the animated props you pass to it. This is what animated is there for, it extends native elements to receive animated values.

See here https://react-spring.io/basics for details